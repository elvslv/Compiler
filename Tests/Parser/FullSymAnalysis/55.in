var i, j: integer;
procedure func(i: integer);
begin
i := i;
end;
begin
for i := func(1) to func(5) do
	j := j + 1;
end.