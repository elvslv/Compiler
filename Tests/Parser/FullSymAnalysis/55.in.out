procedure FUNC(var I : INTEGER;);
begin    
|        
|___:=   
|   |    
|   |___I
|   |    
|   |___I
|        
end      

var I : INTEGER;
var J : INTEGER;

18 7 Init expression must be integer